RELEASES.md

```diff
@@ -0,0 +1,135 @@
+# Releases
+
+This document describes the versioning and release process of OpenFGA. This document will be considered a living document. Scheduled releases, supported timelines, and API stability guarantees will be updated here as they change.
+
+## Release Requirements
+OpenFGA endeavours to rapidly release new features to the community.  To support that goal releases are performed based on the impact and quantity of updates to the codebase in a release-ready state.  Releases may be initiated based on a number of factors.
+
+### Functional Changes
+Changes ready for use in production environments which add new functionality or performance improvements may individually trigger an new release.
+
+### Feature Previews
+New features which are ready for testing by the community but are disabled unless OpenFGA is launched with a feature flag may also trigger a new release.
+
+### Maintenance Updates
+Minor updates, performance improvements, code refinements, or other items which do not introduce signifigant changes may not trigger a release individually, however when several minor changes have been merged a release may be initiated if no new version has been released in two weeks.
+
+### Security Updates and Bugfixes
+Changes made to address a known vulnerability or security issue or fixes to resolve a bug impacting users in production will trigger a release as soon as the changes have been tested and are considered production-ready.  A CVE will also be issued if relevant.
+
+## Versioning
+Releases of OpenFGA will be versioned using dotted triples. For the purposes of this document, we will refer to the respective components of this triple as \<major\>.\<minor\>.\<patch\>. When release candidates are published, they will include the suffix "-rc" followed by a release candidate number which identifies them as "pre-releases".
+
+### Major and Minor Releases
+Changes in the major and minor version components of the triple indicate changes in the functionality or performance of OpenFGA which may include breaking changes requiring those integrating OpenFGA in their project to make changes in their use of OpenFGA when adopting the new version.  It is recommended to fully review the published release notes accompanying a new release before upgrading.
+
+### Patch Releases
+Releases which increment only the "patch" component of the version triple may introduce new functionality, performance enhancements, bugfixes, or preview features while remaining backwards compatible with the previous release.  
+
+## Release History
+Release dates for all releases (not including release candidates)
+
+| Version | Release Date |
+|---------| ------------ |
+| v1.6.1 | Sep 12, 2024 |
```

Comment:
Slight concern that this list can get very long and easily be out of date.

Reply:
@ryanpq would the suggested change above work?

Reply:
Yes, I'll commit an update to make the change.  This was one concern I had as it would have required adding a step to the release playbook to keep it up to date.